pkg/hook/hook.go

```diff
@@ -10,9 +11,10 @@ import (
 
 // Hook represents a single hook configuration.
 type Hook struct {
-	Method  string      `yaml:"method"`
-	Headers http.Header `yaml:"headers"`
-	Body    string      `yaml:"body"`
+	Method  string            `yaml:"method"`
+	Headers http.Header       `yaml:"headers"`
+	Body    string            `yaml:"body"`
+	Params  map[string]string `yaml:"params"`
```

Comment:
We should use [url.Values](https://godoc.org/net/url#Values) instead. URL params can actually be specified multiple times.